packages/twenty-front/src/hooks/usePageChangeEffectNavigateLocation.ts

```diff
@@ -18,6 +18,10 @@ export const usePageChangeEffectNavigateLocation = () => {
     isMatchingLocation(AppPath.Invite) ||
     isMatchingLocation(AppPath.ResetPassword);
 
+  const isMatchingOpenSignedInRoute = isMatchingLocation(
```

Comment:
not sure to understand this: why isMatchingOpenSignedInRoute and matching PlanRequired?

Also, @etiennejou is refactoring this to display the trial choice screen now so this will conflict

I think @etiennejou should include your change in its own PR